apps/core/components/userContext/UserContext.js

```diff
@@ -99,12 +131,28 @@ export default class UserContextProvider extends React.Component<Props, State> {
 
   signOut = async () => {
     try {
-      await firebase.auth().signOut(); // @TODO refactro to GoogleLoginHelper
+      await firebase.auth().signOut();
     } catch (e) {
       // @TODO Handle sign-in error - show notification or message
     }
   };
 
+  setUserPhoneNumber = (phoneNumber: PhoneNumber) => {
+    const { userId } = this.state;
+    const userRef = userId && firebase.database().ref(`users/${userId}`);
```

Comment:
Maybe we could use a helper method here for getting the `phoneNumber` reference and also use it on `getPhoneNumberFromFirebase`, something like:
```
getPhoneNumberReference = (userId: ?string) => {
  return userId && firebase.database().ref(`users/${userId}/phoneNumber`);
}
```

Reply:
good idea 👍

Reply:
done